provision/builders/vmBuilder.go

```diff
@@ -0,0 +1,112 @@
+package builders
```

Comment:
Thanks for the effort, but did u even try to use tfuser ? as I mentioned that tools is not maintained and was built only during the development of the explorer. now the only official client for the explorer is jsng.

Reply:
Removed it and all the additions in old stuff.